### ISSUE TITLE
🐛 internalError/io: "You must have at least one workspace"

### ISSUE THREAD

**Issue by scruffymongrel** [biomejs/biome#4804]
### Environment information

```block
CLI:
  Version:                      1.9.4
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           unset
  JS_RUNTIME_NAME:              unset
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 false

Workspace:
  Open Documents:               0
```

### What happened?

1. configure zed with biome support
2. save a JS file with a deliberate formatting error
3. see message that formatting failed and open the issue
4. see the following message...

```
Language server error: biome

oneshot canceled
-- stderr--
Biome encountered an unexpected error

This is a bug in Biome, not an error in your code, and we would appreciate it if you could report it to https://github.com/biomejs/biome/issues/ along with the following information to help us fixing the issue:

Source Location: crates/biome_service/src/settings.rs:78:14
Thread Name: tokio-runtime-worker
Message: You must have at least one workspace.

internalError/io ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Errors occurred while executing I/O operations.
    
    Caused by:
      the server process exited before the connection could be established
  

```

### Expected result

Formatting should be corrected on save.

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by scruffymongrel** [comment]
Unfortunately I'm not sure how to create a repro as the issue arose inside of Zed.

If anyone can provide advice on how best to create a repro for this kind of issue, I'll see what I can do.

**Comment by scruffymongrel** [comment]
In case it helps, I also had `biome` installed via `brew install biome`... I've since uninstalled it and this issue no longer arises (I still have issues, but I think they're prolly Zed-specific).

**Comment by m1guer** [comment]
you fixed?
